packages/server/test/helpers.ts

```diff
@@ -70,11 +70,13 @@ export const actions = {
 
     this.fork(createOrchestrator({
       delegate,
-      atom: this.atom,
+      atom,
       project: merge(options, overrides || {}),
     }));
-
     await this.receive(delegate, { status: 'ready' });
+    // await this.fork(atom.once(state => {
+    //   return state.commandServer.type === 'started'
+    // }))
```

Comment:
Should this be removed?

Reply:
These were some of the things Charles and I were trying out to debug the issue to get the CLI tests to pass.